src/Gui/PythonWrapper.cpp

```diff
@@ -576,6 +602,33 @@ QGraphicsObject* PythonWrapper::toQGraphicsObject(PyObject* pyPtr)
     return nullptr;
 }
 
+QGraphicsObject* PythonWrapper::toQGraphicsObject(const Py::Object& pyobject)
+{

```

Comment:
I suggest to avoid the code-duplication and instead write: return toQGraphicsObject(pyobject.ptr());

Reply:
Also thanks and done.